next.clayui.com/content/docs/components/tabs.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "Tabs"
+description: "Tabs organize similar content together into individual sections in the same page."
+packageNpm: "@clayui/tabs"
+packageVersion: "3.0.0"
+packageStatus: "Stable"
+sibling: "docs/components/css-tabs.html"
+---
+
+import {
+    Tabs, TabsDropdown
+} from '../../../src/components/clay/Tabs';
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+- [Composing](#composing)
+- [API](#api)
+  - [Item](#item)
+  - [Content](#content)
+  - [TabPane](#tabpane)
+
+</div>
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/tabs/">Tabs Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+<div class="clay-site-alert alert alert-warning">
+	<strong class="lead">Warning</strong>
+	Before you get started with this doc, read about <a href="/foundations/composing.html" target="_blank">Clay's fundamentals of composition</a> to move on. The information below is assuming you have read about <a href="/foundations/composing.html" target="_blank">Clay's compositional philosophy</a> and learned about the terms.
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	We recommend that you review the <a href="https://storybook.clayui.com/?path=/story/components-claycard--claycardwithinfo" target="_blank">use cases in the Storybook</a>.
+</div>
+
+## Composing
+
+`ClayTabs` is a [uncontrolled component](https://reactjs.org/docs/uncontrolled-components.html), we left this component uncontrolled because the user can use Dropdowns or another control to select a `TabPane` for render.
```

Comment:
I don't think I would consider ClayTabs as "uncontrolled." Uncontrolled is something that internally handles the value and on change. Since we require props like `onClick`, `active`, and `activeIndex` it would really be controlled. We can probably leave this out for the user though since its not important for them to know controlled vs uncontrolled for this type of component. So I think we can remove this portion.

Reply:
Sorry, I switched the nomenclature in my mind. You're right. It's a controlled component 😅